README.md

```diff
@@ -176,7 +176,7 @@ Reporters provide information about the current collection run in a format that
   Specify one reporter name as `string` or provide more than one reporter name as a comma separated list of reporter names. Available reporters are: `cli`, `json`, `html` and `junit`.<br/><br/>
 Spaces should **not** be used between reporter names / commas whilst specifying a comma separted list of reporters. For instance:<br/><br/>
 :white_check_mark: `-r html,cli,json,junit` <br/>
-:x: `-r html, cli , json,junit`
+:x: `-r html, cli, json, junit`
```

Comment:
@r3nya This is intentional, as it is meant to highlight an example of incorrectly using comma separated reporter names.

Reply:
@kunagpal fixed! :)